crates/ruma-state-res/.clippy.toml

```diff
@@ -0,0 +1,10 @@
+msrv = "1.50"
+disallowed-types = []
+enforced-import-renames = [ { path = "serde_json::Value", rename = "JsonValue" } ]
+standard-macro-braces = [
+    { name = "btreeset", brace = "[" },
+    # The macro calls itself like btreemap!(...) so this triggers for any use
```

Comment:
I think it would be nice if the lint didn't trigger on macro calls generated by other macro calls. Clippy often filters out code generated by macros by looking at the span AFAIK, maybe the same thing is possible here. And maybe the `maplit` crate would also accept a PR changing this (if they use `hashmap` with curly braces internally, why not `btreemap`?).